### DISCUSSION TITLE
Confusing dashboard setup

### DISCUSSION THREAD

**Discussion by oneart-dev** [SigNoz/signoz#1613] [Q&A]
Thank you for this really impressive app, really like it so far.
But with dashboards i'm totally lost.

Simple example, i have important job on server and it can take any amount of time because of thousands of requests to third party. I want to monitor average duration and if possible send alerts when its above threshold.

For example (see screenshot below), this is perfect chart i want to see exactly copy of it in dashboard. I guess i need to use query like request but don't know where to look for proper explanation of database structure.

<img width="1160" alt="Screenshot 2022-10-05 at 16 10 22" src="https://user-images.githubusercontent.com/70148117/194164389-2ad761a3-9b9b-494a-ab24-d822c0265c0e.png">

Important is to filter by operation and server name. And would be super cool to group by custom tag attached to the trace.

**Comment by pranay01** [comment]
hi @oneart-dev You should be able to create similar chart in dashboards and alerts. 

Here's a demo for metrics query builder - https://youtu.be/u2PiWKEdjCw?t=669

Here's a demo on how to set alerts - https://youtu.be/oQFMfEc9JNI?t=332

Do let me know if you are still facing issues in configuring metrics/alerts. Will be happy to help

**Reply by pranay01** [reply]
You can also join out slack community (https://signoz.io/slack) and we should be able to help you out with any specific question

**Reply by oneart-dev** [reply]
I can't find suitable metric for duration of trace. I see latency but its irrelevant in our case as it not http call but internal task.
